R/entry_functions.R

```diff
@@ -97,3 +97,30 @@ get_entry_picks <- function(entryid = NULL, gw = NULL){
   return(picks2)
 }
 
+get_entry_captain <- function(entryid = NULL, gw = NULL, player_names = FALSE){
```

Comment:
I would remove and default the player_names functionality - it´s a nice touch, and should be useful in the vast majority of cases; the get_player_name() function is light enough that this should not add too much to the load.
```suggestion
get_entry_captain <- function(entryid = NULL, gw = NULL){
```